Exploration Queries/InputEntity_Account/UserAccount_SubsOwnedViaSPNs.yaml

```diff
@@ -0,0 +1,35 @@
+Id: 9c45beeb-5155-4326-8c76-25baf3fa2f6f
+DisplayName: "Subscriptions transitively owned by the user via Azure Service Principals"
+Description: "Returns all subscriptions owned transitively by the user via Azure Service Principals"
+InputEntityType: Account 
+InputFields:
+  - AadUserId 
+OutputEntityTypes:
+  - AzureResource  
+QueryPeriodBefore: 7d 

```

Comment:
Normally we want only 12h before and 12h after, 7d lookback in an expansion query but cause the graph to hang